web/components/shared/themed/themedAdvancedFilters.tsx

```diff
@@ -88,6 +93,24 @@ export function AdvancedFilters({
           Add Filter
         </button>
       </div>
+      <div className="flex flex-row w-full items-end justify-end">
+        {filterLayouts && filters.length > 0 && (
```

Comment:
We should show this button here by default (no ternary check). On the onClick below is where we should check if the filters length > 0. If it isn't, we shoot up an error notification saying `Saved Filters can not be empty`. If it is, then call the `filterLayouts?.onSaveFilters(true)`